DEV-GUIDE.md

```diff
@@ -124,6 +124,13 @@ minikube start \
 
 To enable RBAC, add `--bootstrapper kubeadm --extra-config=apiserver.Authorization.Mode=RBAC` to `minikube start` command, in addition to the flags above.
 
+Note: If you're using newer kubernetes (>= v1.8.0) with kubeadm, the extra-config
+works differently for kubeadm and you also don't need the RBAC flag and the plugin
+names since they should all be enabled by default. Instead use the following command:
```

Comment:
What about 'If you are using kubernetes with version >= v1.8.0 for kubeadm, the extra-config and RBAC flags will be enabled by default. Explicitly enabling them will actually cause errors. Please run:'   ?

Reply:
Done.